CHANGELOG.md

```diff
@@ -9,8 +9,11 @@
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* None.
- 
+* In cases where the main thread would exit before other threads, we could destroy a mutex
+  while it was still in use on other threads. This would cause a crash during shutdown.
+  This bug was introduced by https://github.com/realm/realm-core/pull/3185, which was part
```

Comment:
@finnschiermer Please just use the template (2 lines above) for mentioning when this was introduced.